### ISSUE TITLE
EDL cmx_3600 adapter does not support unnamed markers.

### ISSUE THREAD

**Issue by elabrosseRodeofx** [PixarAnimationStudios/OpenTimelineIO#594]
This is the content of an cmx_3600 EDL exported from Avid media composer:
```
TITLE:   Untitled Sequence.01 
FCM: NON-DROP FRAME
001  103043_0 V     C        01:00:00:00 01:00:08:10 01:00:00:00 01:00:08:10 
* FROM CLIP NAME:  MYMEDIA.MOV 
* LOC: 01:00:02:01 RED
```
As you can see the marker does not have a name

Current behavior:

```
>>> timeline = otio.adapters.read_from_file('markers.edl')
>>> list(timeline.each_clip())[0].markers
[]
```

Expected behavior:

```
>>> timeline = otio.adapters.read_from_file('markers.edl')
>>> list(timeline.each_clip())[0].markers
[
otio.schema.Marker(name='', marked_range=otio.opentime.TimeRange(start_time=otio.opentime.RationalTime(value=86449, rate=24), duration=otio.opentime.RationalTime(value=0, rate=1)), metadata={'cmx_3600': {'color': u'RED'}})
]
```

From what I saw this is due to the regular expression that matches locator comments in cmx_3600.py line 457.
Should OpenTimelineIO support unnamed markers ? I feel like it should since some editing systems are able to export unnamed markers. Let me know what you think !

My corporation is waiting for confirmation that our CLA was received. We will submit a PR as soon as possible.

**Comment by elabrosseRodeofx** [comment]
Made a PR:

https://github.com/PixarAnimationStudios/OpenTimelineIO/pull/598

**Comment by jminor** [comment]
Fixed by #598
